cmd/main.go

```diff
@@ -0,0 +1,53 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+	"time"
+
+	"github.com/Xa4y322/go_final_project/database"
+	"github.com/Xa4y322/go_final_project/handlers"
+
+	"github.com/go-chi/chi/v5"
+)
+
+func getPort() string {
+	const default_port = "7540"
```

Comment:
в го принят CamelCase

```suggestion
	const defaultPort = "7540"
```